data/md/legislature/Jessica-Feldmark-03575039-449a-4378-a021-ebdd11a107f4.yml

```diff
@@ -0,0 +1,15 @@
+id: ocd-person/03575039-449a-4378-a021-ebdd11a107f4
+name: Jessica Feldmark
+given_name: Jessica
+family_name: Feldmark
+image: http://mgaleg.maryland.gov/2023RS/images/feldmark01.jpg
+party:
+- name: Democratic
```

Comment:
@sroomf this person should have a file already and just needs added:
- type: lower
  jurisdiction: ocd-jurisdiction/country:us/state:md/government
  district: 12A
They previously served in house district 12 from jan 2019 until 01/11/2023